core/src/main/resources/schema/infinispan-config-15.0.xsd

```diff
@@ -1230,6 +1237,38 @@
               </xs:documentation>
             </xs:annotation>
           </xs:attribute>
+          <xs:attribute name="container" type="xs:boolean" default="${Tracing.container}">
```

Comment:
I apologize for not commenting on this earlier, but instead of all these boolean attributes, I'd prefer a list :

```xml
<xs:simpleType name="tracing-list">
      <xs:annotation>
         <xs:documentation>
            <![CDATA[
                
            ]]>
         </xs:documentation>
      </xs:annotation>
      <xs:list>
         <xs:simpleType>
            <xs:restriction base="xs:token">
               <xs:enumeration value="container"/>
               <xs:enumeration value="cluster"/>
               <xs:enumeration value="persistence"/>
               <xs:enumeration value="security"/>
               <xs:enumeration value="xsite"/>
            </xs:restriction>
         </xs:simpleType>
      </xs:list>
   </xs:simpleType>
```

so that we can have something like:

```xml
<distributed-cache ...>
   <tracing components="container persistence"/>
</distributed-cache>
```

Reply:
it is OK, you're on time, since we haven't merged it yet :P

Reply:
Looks like we still need this done.

Reply:
I'll push it as soon as I finished...

Reply:
Just pushed!